### PR TITLE
BOAC-42 Add authorized users DB table & seed data

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/BOAC-42

### CODE REVIEW

File: boac/factory.py [hunk 1]
```diff
@@ -10,8 +10,8 @@ def create_app():
     app = Flask(__name__.split('.')[0])
 
     load_configs(app)
-    initialize_db(app)
     initialize_logger(app)
+    db.init_app(app)
```
Comment (raydavis) [inline boac/factory.py]: Relocated to enable logging ASAP.

File: run.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from boac.factory import create_app
 
+app = create_app()
```
Comment (raydavis) [inline run.py]: This enables use of `run.py` as our `FLASK_APP`.

File: boac/models/authorized_user.py [hunk 3]
```diff
@@ -4,32 +4,35 @@
...
+from boac.models.base import Base
 from flask_login import UserMixin
 
-MockedUser = namedtuple('MockedUser', 'uid is_admin is_director is_advisor')
 
+class AuthorizedUser(Base, UserMixin):
+    __tablename__ = 'authorized_users'
+
+    id = db.Column(db.Integer, nullable=False, primary_key=True)
+    uid = db.Column(db.String(255), nullable=False)
```
Comment (johncrossman) [inline boac/models/authorized_user.py]: @raydavis You're playing it safe by forcing `uid` as Integer? Makes sense.

File: tests/test_models/test_authorized_user.py [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
-        '''returns None to Flask-Login for unrecognized UID'''
+    def test_load_unknown_user(self, db_session):
+        """returns None to Flask-Login for unrecognized UID"""
         unknown_uid = 'Ms. X'
         assert subject.load_user(unknown_uid) is None
 
-    def test_load_admin_user(self):
-        '''returns authorization record to Flask-Login for recognized UID'''
+    def test_load_admin_user(self, db_session):
+        """returns authorization record to Flask-Login for recognized UID"""
```
Comment (johncrossman) [inline tests/test_models/test_authorized_user.py]: @raydavis I will add `tests/` to our lint-py setup

File: boac/models/authorized_user.py (hunk 3 above)
Comment (raydavis) [reply boac/models/authorized_user.py]: Yes, I think it's wise to keep UID an opaque string so far as anyone but CalNet is concerned. I believe that's how CS stores it, too.

**Comment by johncrossman** [comment]
@raydavis Looks good. Travis has a complaint.  @paulkerschen should also review

**Comment by paulkerschen** [comment]
Leaving aside Travis's newfound appetite for commas, looks good to me, @raydavis.

**Comment by raydavis** [comment]
Missing trailing comma, of course. :)

BTW, @johncrossman , `tox` from the command line fails for me since I'm still using "conda" instead of "virtualenv". If I decide to stay a maverick, I'll fix it.

**Comment by raydavis** [comment]
The silence of the Travis

**Pull request merged**